packages/discovery-provider/plugins/pedalboard/apps/solana-relay/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@pedalboard/solana-relay",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "start": "ts-node ./src/index.ts",
+    "start:dev": "ts-node-dev ./src/index.ts",
```

Comment:
I would like us to standardize on `dev` as the dev task. It seems to be the standard in most modern repos, including the turborepo examples: https://github.com/vercel/turbo/blob/main/examples/kitchen-sink/package.json

Reply:
nice, can do